+adi/ADIS16480.m

```diff
@@ -56,7 +56,14 @@
         
         function [accelReadings, gyroReadings, magReadings, valid] = stepImpl(obj)
             [dataR, valid] = stepImpl@adi.common.Rx(obj);
-            [accelReadings, gyroReadings, magReadings] = stepAccelGyroMag(obj, dataR);
+            switch length(obj.EnabledChannels)
+                case 6
+                    [accelReadings, gyroReadings] = stepAccelGyro(obj, dataR);
+                    magReadings = null(obj.SamplesPerRead,3);
```

Comment:
I don’t think you want null here, use varargout out  for variable size outputs

Reply:
I implemented varargout here: https://github.com/jpineda3/SensorToolbox/blob/1e816bacb7e49b1af8154e660bb0103c26e73133/%2Badi/ADIS16480.m#L67

This errors "Output argument "varargout{2}" (and maybe others) not assigned during call to "adi/ADIS16480/stepImpl"."